.github/workflows/generate-docs-helm-tests-renovate-pr.yml

```diff
@@ -34,28 +47,20 @@ jobs:
           repo_secrets: |
             APP_ID=mimir-github-bot:app_id
             PRIVATE_KEY=mimir-github-bot:private_key
-      # Generate GitHub App Token
+
+      # Generate GitHub App Token (Directly in the Job That Needs It)
       - name: Generate GitHub App Token
-        id: app-token
+        id: token
         uses: actions/create-github-app-token@v1
         with:
-          app-id: ${{ steps.get-secrets.outputs.APP_ID }}
-          private-key: ${{ steps.get-secrets.outputs.PRIVATE_KEY }}
+          app-id: ${{ env.APP_ID }}
+          private-key: ${{ env.PRIVATE_KEY }}
```

Comment:
do these come into the `env` from the step above? Maybe a tiny comment would help